force-app/main/default/objects/Session__c/Session__c.object-meta.xml

```diff
@@ -159,7 +159,9 @@
         <type>Text</type>
```

Comment:
@bmtemakina I believe this change is related to Session object, is that correct ? if so please remove this metadata form this PR and create another PR for this change.